buildSrc/src/main/kotlin/kotest-publishing-conventions.gradle.kts

```diff
@@ -23,21 +24,41 @@ val ossrhPassword: String by project
 val signingKey: String? by project
 val signingPassword: String? by project
 
+val mavenCentralRepoName = "Deploy"
```

Comment:
Is the case change from "deploy" to "Deploy" intentional?

Reply:
Good spot, yes, it's intentional. 

Gradle uses the repository names to create tasks, and it will capitalize the repo name. So it doesn't matter if the repository is named `deploy` or `Deploy`, the task names will still be `publishFooToDeployRepository`.

Defining the name as `Deploy` consistently saves a few characters below, when filtering based on the task name

```
it.name.endsWith("PublicationTo${mavenCentralRepoName}Repository", ignoreCase = true)
```